doc/source/_static/css/index.css

```diff
@@ -62,7 +62,7 @@
 
 .tab-pane pre {
   margin: 0;
-  padding: 0;
+  padding: 0.5em 1em;
```

Comment:
why the left and right paddings are different? is that intended?

Reply:
If two numbers are specified for the padding property, [the first number applies to top and bottom, the second number applies to left and right](https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax). I just thought it looked a bit nicer this way, but can change it if another spacing would be better.

Reply:
ah, ok. thanks.